Makefile

```diff
@@ -37,8 +37,19 @@ technical-docs:  ## build the technical docs
 guides:	swagger ## build the developer guide docs
 	cd docs/guides; make clean html
 
+# (IS_OPENEDX_TRANSLATIONS_WORKFLOW) is set to "yes" when openedx-translations calls this target.
+# Otherwise, we preserve the old behavior of having segmented po files.
```

Comment:
```suggestion
# (IS_OPENEDX_TRANSLATIONS_WORKFLOW) is set to "yes" when openedx-translations repository extracts translations. 
# Related doc: https://docs.openedx.org/en/latest/developers/how-tos/enable-translations-new-repo.html
# Otherwise, we preserve the old behavior of having segmented po files.
```

Reply:
done